src/makeRobotAccountant.js

```diff
@@ -7,6 +7,18 @@
 
 function makeRobotAccountant() {
   // write code here
+
+  let count = 0;
+
+  function getSum(x) {
+    count++;
+
+    return (y) => {
+      return count <= 3 ? x + y : count % 2 === 0 ? 'Bzzz... Error!' : x + y;
```

Comment:
```suggestion
      return count <= 3 
        ? x + y 
        : count % 2 === 0 
          ? 'Bzzz... Error!' 
          : x + y;
```

Reply:
better to use usual if

Reply:
Good day. Thanks for your advice, well noted.